apps/gro/translations/src/en/email.json

```diff
@@ -0,0 +1,39 @@
+{
+  "content": {
+    "caseworker-intro": "GRO enquiry form has been submitted, with details as follows:",
+    "customer-intro": "Thank you for contacting the General Register Office. We will respond to your enquiry as soon as possible:",
+    "caseworker-outro": "",
+    "customer-outro": ""
```

Comment:
no, this shouldn't be here at all actually - if we aren't translating an outro it shouldn't be in the template - I'll update